test_account.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from account import Account
+from account import *
```

Comment:
Dont do that - https://www.google.com/amp/s/www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/amp/